.github/workflows/pr-labels.yml

```
@@ -0,0 +1,160 @@
+              labels="${labels} ${LABEL}"
+            fi
+          done
+
+          echo "labels=${labels}" >> ${GITHUB_ENV}
+
+      - name: Check Cargo.toml for dependencies changes
+        id: check_cargo_toml
+        run: |
+          # Check if any Cargo.toml files were modified
+          CARGO_TOML_FILES=$(echo "${modified_files}" | grep -E 'Cargo.toml')
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)